test/end-to-end.js

```diff
@@ -1043,6 +1043,66 @@ describe('end-to-end', function() {
         .end(done);
     }
   });
+
+  describe('Multi-project workspace support', function() {
```

Comment:
I am wondering what is the reason for moving this test from `test/workspace` to `test/end-to-end`?

The tests in `end-to-end` are running `npm install` and taking long time to complete, that's why they are separated in a standalone file.

AFAICT, your test only calls `createFromTemplate`, which is relatively fast and there are already other tests calling this method in `test/workspace`.

Can we move this test/describe block back to `test/workspace`?